script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -242,7 +242,7 @@ His swordsmanship... Well, I guess %{i}machetemanship%{/i}, is also much more fo
 [sha:76fb0213067bffa7a5833f8158f84bbdc451004e]{
 -- Page 478, Offset 9870.
 -- もう完全にアルクェイドの速度と動作を見切っているという事実であり、なにより、そう―――
-On top of that, it's clear that he's already able to see right through Arcueid's speed and movements. But most importantly of all―――
+On top of that, it's clear that he's already able to see right through Arcueid's speed and movements. But most important of all―――
```

Comment:
Original is correct. Full sentence would be 'But most importantly [of all], he still hasn't used his 'flaming hand''

<https://oneminuteenglish.org/en/most-importantly-or-most-important/>
<https://www.grammarphobia.com/blog/2019/01/important-importantly.html#:~:text=A%3A%20Either%20%E2%80%9Cmost%20important%E2%80%9D,%E2%80%9Ccorrect%E2%80%9D%20than%20the%20other.>

Reply:
Yusuke is definitely right.